### ISSUE TITLE
Eliminate deprecation warnings emitted by ES container 

### ISSUE THREAD

**Issue by amarjandu** [DataBiosphere/azul#2758]
These deprecation warnings were found being emitted from the container that runs Elasticsearch during unit-tests as part of the changes from #2703. 

Here is a sample of some of the messages that have been emitted:
```
Accessing variable [_agg] via [params._agg] from within a scripted metric agg combine script is deprecated in favor of using [state].

Accessing variable [_aggs] via [params._aggs] from within a scripted metric agg reduce script is deprecated in favor of using [state].

Fielddata access on the _uid field is deprecated, use _id instead.

OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated
```

A full output of the logs during the test run have been attached below:
[deprecation-warnings-es-container.txt](https://github.com/DataBiosphere/azul/files/5922207/deprecation-warnings-es-container.txt)

**Comment by amarjandu** [comment]
Note for the JDK warning, we might opt to not resolve the warning as it comes from the image that we use.  https://github.com/DataBiosphere/azul/blob/80ccf86ccfaebc246e103405b90e4ca9b03e305d/test/es_test_case.py#L39

This might be able to be resolved by changing to the Java G1 Garbage Collector, if this is something that can be changed from the image configuration.

**Comment by melainalegaspi** [comment]
@amarjandu please confirm whether or not you created FIXMEs for this.

**Comment by melainalegaspi** [comment]
Spike to answer the following questions: does this warning occur in the CloudWatch logs for real deployments? If not, why? Have the deprecated artifacts been removed in ES 7?

**Comment by amarjandu** [comment]
Searched through `prod` and `dev` `ES`, `indexer`, `service` log groups using Cloudwatch insights, unable to locate any `deprecated` messages. 
It appears that within ES 6.8 there are configuration options for ES https://www.elastic.co/guide/en/elasticsearch/reference/6.8/logging.html . We might want to ask AWS support which log group we can expect these messages to appear within, they might have changed the configuration for hosted ES instances.

**Comment by amarjandu** [comment]
It looks like if we want to update to ES 7 we have to address breaking changes.

`_uid` https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#uid-meta-field-removed

`use state within script context rather than params._agg` https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#replace-params-agg-with-state-context-variable

**Comment by melainalegaspi** [comment]
@hannes-ucsc : "Spike again to determine if the recent upgrade of the ES container from 6.8.0 to 6.8.20 eliminated the warnings. Also, consider that the disabling of the full manifest eliminated the warnings, since they make use of scripted aggregation, I think."

**Comment by amarjandu** [comment]
6.8.20 still shows deprecated ES warnings.
Full `make test` output https://gitlab.dev.singlecell.gi.ucsc.edu/-/snippets/22

patch used
```
From fab6fe8e6607b6380045df58e113dddae458cdea Mon Sep 17 00:00:00 2001
From: amar jandu <amarjandu@specterbyte.com>
Date: Wed, 5 Jan 2022 09:54:04 -0800
Subject: [PATCH] Turn off ES warning exceptions

---
 test/azul_test_case.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/test/azul_test_case.py b/test/azul_test_case.py
index b564f29a..72bf0d45 100644
--- a/test/azul_test_case.py
+++ b/test/azul_test_case.py
@@ -70,11 +70,11 @@ class AzulTestCase(TestCase):
                 #        https://github.com/DataBiosphere/azul/issues/2070
                 '"@coroutine" decorator is deprecated since Python 3.8, use "async def" instead',
                 # FIXME: https://github.com/DataBiosphere/azul/issues/2758
-                'OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated',
-
-                RE(r'.*Fielddata access on the _uid field is deprecated, use _id instead'),
-                RE(r'.*Accessing variable \[_aggs\]'),
-                RE(r'.*Accessing variable \[_agg\]'),
+                # 'OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated',
+                #
+                # RE(r'.*Fielddata access on the _uid field is deprecated, use _id instead'),
+                # RE(r'.*Accessing variable \[_aggs\]'),
+                # RE(r'.*Accessing variable \[_agg\]'),
 
                 # FIXME: furl.fragmentstr raises deprecation warning
                 #        https://github.com/DataBiosphere/azul/issues/2848
-- 
2.30.2
```

**Comment by amarjandu** [comment]
@hannes-ucsc "Ticket #3006 will deal with the deprecation of the `_uid` field. The `_agg/aggs` deprecation will need to be addressed after #3006"

**Comment by hannes-ucsc** [comment]
Can't reproduce. The warning is not being emitted.
